stack.yaml

```diff
@@ -113,9 +113,12 @@ extra-deps:
 - concurrent-extra-0.7.0.10       # not yet on Stackage
 # - purescript-bridge-0.8.0.1
 - directory-1.3.1.0               # https://github.com/malcolmwallace/cpphs/issues/8
-- servant-0.10                    # servant-multipart supports version servant-10 only
-- servant-server-0.10             # so it triggers another dependencies to be v10
-- servant-multipart-0.10
+- servant-0.12
+- servant-server-0.12             # so it triggers another dependencies to be v10
```

Comment:
`# so it triggers another dependencies to be v10` is now redundant and can be removed 😉